### PR TITLE
[15.0] [MIG] account_move_fiscal_year

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
This module displays the fiscal year on journal entries/items.

### CODE REVIEW

File: account_move_fiscal_year/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Installation
+============
+
+You need to install account_fiscal_year.
+
+Configuration
+=============
+
+You just need to create date ranges associated to 'Fiscal Year' type.
```
Comment (AaronHForgeFlow) [inline account_move_fiscal_year/README.rst]: Is it using date range anymore?

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM Functional tests. Pending to change readme, minor request

BTW: I still think date.range is way better this account.fiscal.year object. But that's a Odoo decision we cannot change :(

File: account_move_fiscal_year/README.rst (hunk 1 above)
Comment (JuanyDForgeflow) [reply account_move_fiscal_year/README.rst]: @AaronHForgeFlow done

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**